docs/reference/logging/journald.md

```diff
@@ -41,6 +42,11 @@ You can set the logging driver for a specific container by using the
 Users can use the `--log-opt NAME=VALUE` flag to specify additional
 journald logging driver options.
 
+### tag
+
+Template to set `CONTAINER_TAG` value in journald logs, refer to
```

Comment:
Hum this one should be a sentence, something like `Specify tag for message, which interpret some markup. Refer to the log tag option documentation for customizing the log tag format.` (borrowed from `splunk.md`).